cypress/reporter-config.json

```
@@ -1,6 +1,6 @@
 {
   "reporterEnabled": "spec, mocha-junit-reporter",
   "mochaJunitReporterReporterOptions": {
-    "mochaFile": "cypress/results/results-[hash].xml"
+    "mochaFile": "cypress/results/express-[hash].xml"
```

Can we maybe name this simorgh instead of express?
```suggestion
    "mochaFile": "cypress/results/simorgh-[hash].xml"
```